x-pack/plugins/session_view/public/components/detail_panel_alert_group_item/index.tsx

```diff
@@ -39,19 +40,19 @@ export const DetailPanelAlertGroupItem = ({
     return null;
   }
 
-  const { rule } = alerts[0].kibana.alert;
+  const { rule } = alerts[0]?.kibana?.alert ?? {};
```

Comment:
can't return earlier if there is no `alerts[0]?.kibana?.alert`?

Reply:
I think it depends on if we want to show all the fields with fall back value (i.e. `-`) instead of returning null 🤔